Source/Fuse.Nodes/Visual.WorldTransformInvalidated.uno

```diff
@@ -0,0 +1,68 @@
+using Uno;
+using Uno.Collections;
+using Uno.UX;
+using Uno.Matrix;
+
+namespace Fuse
+{
+	/*
+		This file implements the WorldTransformInvalidated event and calls the OnInvalidateWorldTransform method, 
+		which due to optimizations can not be generated the intuitive way anymore. 
+		This implementation is hackish, but should work. It takes the cost proportional to the number of consumers
+		of the event, instead of paying a high toll all the time when transforms are invalidated.
+		The events are rarely used, and the optimizations are very important.
+	*/
+
+	public partial class Visual
+	{
+		static List<Visual> _wtiSubjects = new List<Visual>();
+
+		static void BeginWatchWTI()
+		{
+			UpdateManager.AddAction(ProcessWTIHandlers);
```

Comment:
I'd rather see this done in the App class, or in the Rootviewport rooting, than as a static. Statics and testing don't play well together.